client/ayon_core/lib/env_tools.py

```diff
@@ -27,7 +55,11 @@ def env_value_to_bool(env_key=None, value=None, default=False):
     return default
 
 
-def get_paths_from_environ(env_key=None, env_value=None, return_first=False):
+def get_paths_from_environ(
+    env_key: Optional[str] = None,
+    env_value: Optional[str] = None,
+    return_first: bool = False,
+) -> Optional[Union[str, list[str]]]:
     """Return existing paths from specific environment variable.
 
     Args:
```

Comment:
Type hints in the docstring do not match the type hints of the arguments? (The `str` are not optional in the docstring?)